mkosi

```diff
@@ -1374,7 +1374,7 @@ def find_kernel_file(workspace_root, pattern):
     workspace_pattern = os.path.join(workspace_root, pattern.lstrip('/'))
     kernel_files = sorted(glob.glob(workspace_pattern))
     # the path kernel-install expects is within the workspace reference as it is run from within the container
-    kernel_file = '/' + (kernel_files[0].lstrip(workspace_root))
+    kernel_file = kernel_files[0].replace(workspace_root, '')
```

Comment:
This is also fragile if the `workspace_root` appears again in the path later. Some combination of `startswith` and then a splice operator is probably the safest option.

Reply:
Good point, please check my changes